cluster-autoscaler/core/scale_down.go

```diff
@@ -678,16 +659,16 @@ func (sd *ScaleDown) markSimulationError(simulatorErr errors.AutoscalerError,
 // chooseCandidates splits nodes into current candidates for scale-down and the
 // rest. Current candidates are unneeded nodes from the previous run that are
 // still in the nodes list.
-func (sd *ScaleDown) chooseCandidates(nodes []*apiv1.Node) ([]*apiv1.Node, []*apiv1.Node) {
+func (sd *ScaleDown) chooseCandidates(nodes []string) ([]string, []string) {
```

Comment:
Nit: Could you please assign names for returned lists to increase readability?

Reply:
Is this better?

Reply:
I like it much more. Thanks.